### DISCUSSION TITLE
Symbolicated crash reporting

### DISCUSSION THREAD

**Discussion by jvanderwee** [getsentry/sentry-kotlin-multiplatform#120] [Q&A]
We currently use [CrashKiOS](https://crashkios.touchlab.co/) for properly symbolicated Kotlin stack traces in Crashlytics. 

We've yet to try sentry-kotlin-multiplatform. Are Kotlin stack traces symbolicated properly in Sentry for an iOS project using Kotlin Multiplatform?

**Comment by romtsn** [comment]
hi @jvanderwee thanks for your interest in our kmp sdk! Yes, symbolication should work if you follow the steps mentioned [here](https://docs.sentry.io/platforms/kotlin-multiplatform/debug-symbols/). If not, please let us know through opening a new github issue. Thanks!

P.S. This is what it looks like on our KMP sample which you can find in this repo
<img width="926" alt="image" src="https://github.com/getsentry/sentry-kotlin-multiplatform/assets/4999776/8581f725-5668-49bd-8bf4-eb1471f952e8">

**Reply by jvanderwee** [reply]
Awesome! Thank you 🙏
